source/introduction.rst

```diff
@@ -20,6 +45,14 @@ The program `terminal` will launch and you should see a prompt::
 
 The line tells you your current location followed by `>`. 
 
+
+.. tip::
+
+    We want to change directories into the 'documents' folder.
```

Comment:
It might be worth actually saying `folders` or explaining further that directories is just another work for folders